releases/v2.1.0-alpha.20180730.md

```diff
@@ -0,0 +1,263 @@
+---
+title: What&#39;s New in v2.1.0-alpha.20180730
+toc: false
+summary: Additions and changes in CockroachDB version v2.1.0-alpha.20180730 since version v2.1.0-alpha.20180702
+---
+
+## July 25, 2018
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-alpha.20180730.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-alpha.20180730.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-alpha.20180730.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-alpha.20180730.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### General Changes
+
+- The [cost-based optimizer](../v2.1/sql-optimizer) is now on by default. [#26893][#26893]
+- The time series system used by CockroachDB to store internal monitoring data now utilizes pre-computed rollups to significantly increase the duration for which monitoring data is available while using less storage. Monitoring data will be available for up to a year by default; however, data older than seven days will be stored at a reduced resolution, and thus will only be able to give details about 30 minute intervals. [#27121][#27121] {% comment %}doc{% endcomment %}
+- Building CockroachDB from source now requires yarn at version 1.7.0 or above. [#27262][#27262] {% comment %}doc{% endcomment %}
+- Added support for signing server and client certificates by different CAs. [#27636][#27636] {% comment %}doc{% endcomment %}
+
+### Enterprise Edition Changes
+
+- Core dumps are now disabled and [encryption](../v2.1/encryption.html) is enabled. [#27426][#27426]
+- [`CHANGEFEED`](../v2.1/change-data-capture.html)s now use an asynchronous Kafka producer, increasing throughput. [#27421][#27421]
```

Comment:
I would suggest putting the plural "s" between the brackets:

```
- [`CHANGEFEED`s](...) now use ...
```

Reply:
Done.